axon-server/administration/clustering.md

```diff
@@ -0,0 +1,51 @@
+# Clustering
+
+> Note - This feature is only available in Axon Server Enterprise
+
+## Introduction
+
+Axon Server Enterprise can be deployed as a _**cluster**_ to guarantee high availability. A cluster of Axon Server nodes will provide multiple connection points for \(Axon Framework-based\) client applications, and thus share the load of managing message delivery and event storage. Client applications will dynamically connect to a node in the cluster and automatically reconnect to another, should the node that they are currently connected to become unreachable.‌
```

Comment:
So, for now it's Axon Framework-based applications, but this actually depends on the use of the Axon Server API.
If you'd be to built your own in for example Go, you wouldn't be tied to Axon Framework at all. Might be nice to have a short notice on this, but no sure if this is the right spot for it.

Reply:
I can include this in another section...if thats ok

Reply:
Completely fine with that @domaincomponents!